kite-data/kite-data-core/src/main/java/org/kitesdk/data/spi/filesystem/CSVUtil.java

```diff
@@ -42,6 +43,14 @@
 
 public class CSVUtil {
 
+  public static CSVParser newParser(CSVProperties props) {
+    return new CSVParser(
+        props.delimiter.charAt(0), props.quote.charAt(0),
+        props.escape.charAt(0),
+        false /* strict quotes off: don't ignore unquoted strings */,
+        true /* ignore leading white-space */ );
```

Comment:
Not until someone requests them. I don't want to over-complicate the builders, commands, etc. unless people will actually use it. These seem like reasonable defaults.